File: subprojects/frida-node/examples/query_system_parameters.ts
import frida from "frida";

async function main() {
    console.log("Local parameters:", await frida.querySystemParameters());

    const device = await frida.getUsbDevice();
    console.log("USB device parameters:", await device.querySystemParameters());
}

main()
    .catch(e => {
        console.error(e);
    });

File: subprojects/frida-python/examples/query_system_parameters.py
from pprint import pformat

from pygments import highlight
from pygments.formatters import Terminal256Formatter
from pygments.lexers import PythonLexer

import frida

print("Local parameters:", highlight(pformat(frida.query_system_parameters()), PythonLexer(), Terminal256Formatter()))
print(
    "USB device parameters:",
    highlight(pformat(frida.get_usb_device().query_system_parameters()), PythonLexer(), Terminal256Formatter()),
)
